podium/talks/views.py

```diff
@@ -4,8 +4,6 @@
 from .forms import TalkSubmissionForm
 from .models import Talk, Session
 
-
```

Comment:
Please do not commit only whitespace changes.  Use `git commit --amend` to fix whitespace issues.